src/collectors/diskspace/diskspace.py

```diff
@@ -146,7 +146,8 @@ def get_file_systems(self):
                      mount_point.startswith('/sys'))):
                     continue
 
-                if '/' in device and mount_point.startswith('/'):
+                if ((('/' in device or device == 'tmpfs') and
```

Comment:
So if the conf is modified to allow tmpfs as well, stats for /tmp will also
go through.

On Thu, May 5, 2016 at 12:02 PM, Grant Ridder <notifications@github.com>
wrote:

> In src/collectors/diskspace/diskspace.py
> <https://github.com/python-diamond/Diamond/pull/471#discussion_r62237019>:
>
> > @@ -146,7 +146,8 @@ def get_file_systems(self):
> >                       mount_point.startswith('/sys'))):
> >                      continue
> >
> > -                if '/' in device and mount_point.startswith('/'):
> > +                if ((('/' in device or device == 'tmpfs') and
>
> Gotcha, tmpfs is still skipped though and won't make it to this line
> since it is not in filesystems
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/python-diamond/Diamond/pull/471/files/17d872376ee741c1f49a6e67c06a9a5cd7f2138d#r62237019>
>